kong/clustering/rpc/socket.lua

```diff
@@ -178,8 +178,9 @@ function _M:process_rpc_msg(payload, collection)
 
       -- collection is nil, it means it is a single call
       -- we should call async function
-      local name = string_format("JSON-RPC callback for node_id: %s, id: %d, method: %s",
-                                 self.node_id, payload_id or 0, payload_method)
+      -- random is for avoiding timer's name confliction
+      local name = string_format("JSON-RPC callback for node_id: %s, id: %d, rand: %d, method: %s",
+                                 self.node_id, payload_id or 0, math.random(10^5), payload_method)
```

Comment:
What is the point of named timer, if it is random? Can we just use unnamed timer?

Reply:
The named timer should be good for troubleshooting.